lib/config.js

```diff
@@ -2,15 +2,20 @@ const { validateSchema } = require('./schema')
 const { DEFAULT_CONFIG } = require('./default-config')
 const log = require('./log')
 
-const CONFIG_NAME = 'release-drafter.yml'
+const DEFAULT_CONFIG_NAME = 'release-drafter.yml'
```

Comment:
I think this should move to the `default-config.js` 🤔

Reply:
The responsibilities seem to be good at the moment:

- `default-config.js`: export a default configuration object
- `config.js`: load and validate a configuration, falling back to a default one
- `index.js`: the controller - it asks `config.js` for the configuration to use, passing the context and the config name if present on the input

It doesn't make much sense to me that `index.js` will import `default-config.js` just to pass something to `config.js` that it already has access to (because it's already importing it).

Reply:
As I said minor nitpick 😄

Reply:
Alright then! 👍
Let's keep this conversation open, someone else might want to give an opinion.